supply-chain-security/compromises/2019/purescript-npm.md

```diff
@@ -0,0 +1,12 @@
+# Malicious code in purescript npm installer dependencies
+Malicious code was inserted in dependencies of the purescript npm installer. 
+The code was inserted in the packages load-from-cwd-or-npm and rate-map.
+
+
+## Impact
+Both packages have ~ 1400 downloads weekly.
+
+
+## Type of compromise
+The maintainer at that time claims that the malicious code was published by an attacker 
```

Comment:
Can you reword this to follow the wording style of other compromises. "It appears ..." when uncertain